properties_pane/field_level/fieldLevelConfig.json

```diff
@@ -610,7 +615,18 @@ making sure that you maintain a proper JSON format.
 					]
 				}
 			},
-			"default",
+			{
+				"fieldName": "Default",
+				"fieldKeyword": "default",
+				"shouldValidate": true,
+				"fieldType": "text",
+				"regex": "(?:\\s|\\d|\\S)+"
```

Comment:
Added custom default property validation for number type because this property in addition to supporting number values also supports default procedures like aggregation functions